### ISSUE TITLE
split.table is not working

### ISSUE THREAD

**Issue by Quiri** [Rapporter/pander#164]
The value supplied to split.table is not working correct. It already splits if the width is less. In my case it splits a 75 char table with split.table = 90. It doesn't split if i set the value to >95.

**Comment by RomanTsegelskyi** [comment]
@Quiri, can you please give a reproducible example? That will make it easier to find out what is wrong

**Comment by daroczig** [comment]
I suspect some Unicode character issue. @Quiri, although I suppose you do, but can you please verify that you are using the dev version of `pander`? And please post some reproducible example as Roman suggested.

**Comment by daroczig** [comment]
Okay, I tried to reproduce it with Unicode chars, quick example:

```r
> pander(data.frame(a = '7 chars', b = paste(rep('Δ', 5), collapse = ' ')), split.table = 24)

-----------------
   a        b    
------- ---------
7 chars Δ Δ Δ Δ Δ
-----------------
```

But:

```r
> pander(data.frame(a = '7 chars', b = paste(rep('Δ', 5), collapse = ' ')), split.table = 23)

-------
   a   
-------
7 chars
-------

Table: Table continues below

 
---------
    b    
---------
Δ Δ Δ Δ Δ
---------
```

How that comes?

```r
> nchar(paste(rep('Δ', 5), collapse = ' '), type = 'width')
[1] 9
> nchar('7 cahrs', type = 'width')
[1] 7
```

So that's exactly 16 chars. But we split until up to 23 chars, so we need at least `split.table=24` not to split it. And this is not due to Unicode stuff as I first thought, but we are comparing column width **plus an extra 4 chars per columns** to the `split.table` argument in [pandoc.R#L985](https://github.com/Rapporter/pander/blob/master/R/pandoc.R#L985).

This code chunk was not touched for more than 2 years -- maybe it's time to review that :)

**Comment by RomanTsegelskyi** [comment]
Fixed in #186
